packages/lab/src/overlay/OverlayPanel.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  ComponentPropsWithoutRef,
+  ForwardedRef,
+  forwardRef,
+  HTMLProps,
+} from "react";
+import { makePrefixer, useFloatingComponent, useForkRef } from "@salt-ds/core";
+import { clsx } from "clsx";
+import { useOverlayContext } from "./OverlayContext";
+import { FloatingOverlay } from "@floating-ui/react";
+import { OverlayPanelBase } from "./OverlayPanelBase";
+
+const withBaseName = makePrefixer("saltOverlayPanel");
+
+export interface OverlayPanelProps extends ComponentPropsWithoutRef<"div"> {}
+
+export const OverlayPanel = forwardRef<HTMLDivElement, OverlayPanelProps>(
+  function OverlayPanel(props, ref: ForwardedRef<HTMLDivElement>) {
+    const { className, ...rest } = props;
+
+    const { Component: FloatingComponent } = useFloatingComponent();
+
+    const {
+      id,
+      openState,
+      floatingStyles,
+      placement,
+      context,
+      getFloatingProps,
+      floating,
+    } = useOverlayContext();
+
+    const handleRef = useForkRef<HTMLDivElement>(floating, ref);
```

Comment:
minor suggestion, but think this is more readable
```suggestion
    const overlayPanelRef = useForkRef<HTMLDivElement>(floating, ref);
```

Reply:
Several other components seem to have been named in similar format - will be keeping it as is for now and update all components as and when needed